real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/HighTrustGPSLocalizer.java

```diff
@@ -18,44 +20,58 @@
  *
  */
 public class HighTrustGPSLocalizer implements Node{
-    private double buggyFrameGpsLon;
-    private double buggyFrameGpsLat;
+    private double buggyFrameGpsX;
+    private double buggyFrameGpsY;
     private double buggyFrameRotZ;
+    private Date mostRecentUpdate;
     //private double roll = 0;
     //private double pitch = 0;
 
+    private static final double FEET_TO_METERS = 0.3048;
 
     private Publisher posePub;
+    private double lastEncoderMeasurement;
 
     /**
      * Constructor for the High Trust Localizer which will initialize the system to an identity (zero position)
      */
     public HighTrustGPSLocalizer(){
         //init values
-        buggyFrameGpsLon = 0.0;
-        buggyFrameGpsLat = 0.0;
+        buggyFrameGpsX = 0.0;
+        buggyFrameGpsY = 0.0;
+        buggyFrameRotZ = 0.0;
         posePub = new Publisher(NodeChannel.POSE.getMsgPath());
-
+        mostRecentUpdate = new Date();
 
 
         //Initialize subscriber to GPS measurements
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2337031330/issues/source?bid=2938665&fileBranchId=3199623#l55)